### DISCUSSION TITLE
Google Password Manager not working on I3wm

### DISCUSSION THREAD

**Discussion by lk-kheir** [i3/i3#5133] [Q&A]
## Current Behavior
I insattalled i3wm on Kubuntu 22.04 , everything went ok but I noticed later that after I logged in to my google account on Google Chrome version 105.0.5195.125 (Official Build) (64-bit) that all my saved passwords are gone. although I did not have the same problem before insatlling i3wm.
## Expected Behavior
I exprect chrome to show all my passwrds. What should I do to fix this bug.
# Environment
i3 version: 4.20.1

**Comment by ankurdhama** [comment]
Looks like your passwords are saved in kwallet (KDE wallet) and this will not get auto enabled by i3 (i3 is just a window manager).
Please have a look at https://wiki.archlinux.org/title/KDE_Wallet to how to auto unlock kwallet on login.
